cli/cmd/component-delete.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	"github.com/kinvolk/lokomotive/pkg/components"
 	"github.com/kinvolk/lokomotive/pkg/components/util"
+	"github.com/kinvolk/lokomotive/pkg/config"
```

Comment:
I think all the component delete code can be coalesced into one commit. It is very confusing to read some change here and some change there. I think it makes sense to add them all to one commit and then list all the changes that have happened.

Reply:
If you want to review changes aggregated, you can use `git diff` with range or "Changed files" tab in GitHub UI. I wouldn't lose the information of those changes, as each of them is a small independent step making things a bit better.

Also, having more commits makes bisecting easier in case of bugs, which definitely might be introduced by those changes.